app/src/main/java/com/example/simplecaclulator/CalculatorFragment.kt

```diff
@@ -0,0 +1,163 @@
+package com.example.simplecaclulator
+
+import android.annotation.SuppressLint
+import android.os.Bundle
+import android.util.Log
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Button
+import android.widget.TextView
+import androidx.fragment.app.Fragment
+import com.example.simplecaclulator.databinding.FragmentCalculatorBinding
+
+class CalculatorFragment : Fragment() {
+    private var _binding: FragmentCalculatorBinding? = null
+
+    private val binding get() = _binding!!
+
+    private val workingsTV: TextView
+        get() = binding.workingsTV
+    private val resultsTV: TextView
+        get() = binding.resultsTV
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+        _binding = FragmentCalculatorBinding.inflate(inflater, container, false)
+        return binding.root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+
+        binding.apply {
+            clearButton.setOnClickListener { clearAction() }
+            backspaceButton.setOnClickListener { backspaceAction() }
+            addDecimalButton.setOnClickListener { addDecimal(it) }
+            equalsButton.setOnClickListener { equalsAction() }
+
+            //number buttons
+            Button1.setOnClickListener { numberAction(it) }
```

Comment:
give better namings for these buttons. use appropriate codestyle(start word from lowercase)